frontend/src/metabase/writeback/components/ActionCreator/FormCreator/FormField.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { t } from "ttag";
+import { InputType } from "metabase/writeback/types";
+
+import Input from "metabase/core/components/Input";
+import Select from "metabase/core/components/Select";
+import FormTextArea from "metabase/admin/datamodel/components/FormTextArea";
+import NumericInput from "metabase/core/components/NumericInput";
+import DateInput from "metabase/core/components/DateInput";
+import Radio from "metabase/core/components/Radio";
+
+const sampleOptions = [
+  { name: t`Option One`, value: 1 },
+  { name: t`Option Two`, value: 2 },
+  { name: t`Option Three`, value: 3 },
+];
+// sample form fields
+export function FormField({ type }: { type: InputType }) {
```

Comment:
This is very much a placeholder to demonstrate the basic functionality of sample inputs populating. We probably want to come up with some more robust logic that is shared between the action creator and the action form generation component for dashboards.

I wonder if there's a way to use some of `WritebackForm` - unfortunately that relies heavily on field metadata that we don't have access to for SQL queries.

Reply:
Pointed you to that already, but [`ActionParameterInputForm`](https://github.com/metabase/metabase/blob/master/frontend/src/metabase/writeback/containers/ActionParametersInputForm.tsx) can poteintally be reused to some extent